<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCLanguage</class>
 <widget class="QWidget" name="VSCLanguage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Master Hard Disk Selecting</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/logo/resources/vscsmall.jpg</normaloff>:/logo/resources/vscsmall.jpg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <item row="4" column="1">
        <widget class="QRadioButton" name="radioButtonZH">
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
    image: url(:/action/resources/radiobutton_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/action/resources/radiobutton_unchecked_hover.png);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/action/resources/radiobutton_unchecked_pressed.png);
}

QRadioButton::indicator::checked {
    image: url(:/action/resources/radiobutton_checked.png);
}

QRadioButton::indicator:checked:hover {
    image: url(:/action/resources/radiobutton_checked_hover.png);
}

QRadioButton::indicator:checked:pressed {
    image: url(:/action/resources/radiobutton_checked_pressed.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>English</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>简体中文</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="radioButtonEN">
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
    image: url(:/action/resources/radiobutton_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/action/resources/radiobutton_unchecked_hover.png);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/action/resources/radiobutton_unchecked_pressed.png);
}

QRadioButton::indicator::checked {
    image: url(:/action/resources/radiobutton_checked.png);
}

QRadioButton::indicator:checked:hover {
    image: url(:/action/resources/radiobutton_checked_hover.png);
}

QRadioButton::indicator:checked:pressed {
    image: url(:/action/resources/radiobutton_checked_pressed.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Auto</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioButtonAuto">
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
    image: url(:/action/resources/radiobutton_unchecked.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/action/resources/radiobutton_unchecked_hover.png);
}

QRadioButton::indicator:unchecked:pressed {
    image: url(:/action/resources/radiobutton_unchecked_pressed.png);
}

QRadioButton::indicator::checked {
    image: url(:/action/resources/radiobutton_checked.png);
}

QRadioButton::indicator:checked:hover {
    image: url(:/action/resources/radiobutton_checked_hover.png);
}

QRadioButton::indicator:checked:pressed {
    image: url(:/action/resources/radiobutton_checked_pressed.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/button.png);}
QPushButton:hover {
	background-image: url(:/action/resources/buttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/button.png);}
</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/button.png);}
QPushButton:hover {
	background-image: url(:/action/resources/buttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/button.png);}
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../main/vscmainwindows.qrc"/>
 </resources>
 <connections/>
</ui>
